### DISCUSSION TITLE
"filtering attributes" and "convert" mentioned in readme?

### DISCUSSION THREAD

**Discussion by jrochkind** [okuramasafumi/alba#231] [Q&A] [answered]
The [README section on "conditional attributes"](https://github.com/okuramasafumi/alba#conditional-attributes) says:

> Filtering attributes with overriding `convert` works well for simple cases. 

and

> Below is an example for the same effect as [filtering attributes section](https://github.com/okuramasafumi/alba#filtering-attributes).

But the link on  "filtering attributes section" doesn't work, I can't find a "filtering attributes" section of the README. And I can't find any references to "overriding `convert`" (which perhaps were in that section?)

I am interested in additional options for "filtering attributes" , what are these references? Thanks!

**Comment by okuramasafumi** [comment]
Turned into an issue at #235

**Reply by okuramasafumi** [reply, marked as answer]
Hi @jrochkind the commit bbac0447a00e9eec58e203e7e2c4ac53b0393ed4 should fix the link. As for `convert` method, now that we have `attributes` method for overriding we don't need to override `convert` method, so I removed it from the doc.
Do you have something you want to add to README? If not, I'll close #235

**Reply by jrochkind** [reply]
OK, thanks!  That does seem to make the docs consistent!
